kuka_kr210_support/launch/load_kr210l150-2.launch

```diff
@@ -0,0 +1,3 @@
+<launch>
```

Comment:
I would rename this file to `load_kr210l150_2.launch` (i.e., no dashes)

Reply:
This file is non-existent now.